src/routes/index.js

```diff
@@ -22,6 +23,7 @@ router.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerSpec));
 router.use('/arts', artsRoute);
 router.use('/artsreport', reportRouter);
 router.use('/arts/comments/', commentRouter);
+router.use('/comments', commentReaction);
```

Comment:
How's this different from the one above it?

Reply:
the routes are different, `/comments/:id/like` to like a specific comment, and `/arts/comments` to comment on an art